docs/src/main/sphinx/release/release-435.md

```diff
@@ -0,0 +1,117 @@
+# Release 435 (6 Dec 2023)
+
+## General
+
+* Fix possible query failure for `MERGE` queries when `retry-policy` set to
+  `TASK` and `query.determine-partition-count-for-write-enabled` set to `true`.
+  ({issue}`19979`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* {{breaking}} Remove support for registering external tables with
+  `CREATE TABLE` and the `location` table property. Use the
+  `register_table` procedure as replacement. The property
+  `delta.legacy-create-table-with-existing-location.enabled` is
+  also removed. ({issue}`17016`)
+* Ensure AWS access keys are used for connections to the AWS Security Token
```

Comment:
@grantatspothero .. how about this? Also .. find it in all 4 connectors .. let me know if this is correct.

Reply:
Yes that description sounds good, thank you!

It applies to all 4 connectors, but only when `fs.native-s3.enabled` is true. By default, that is false.

Reply:
I assume that is documented .. if not .. can you send a PR?